CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.99.0
+- The following steps were deprecated and will be removed in the upcoming major version:
+  - `/^the "([^"]*)" field should have no error$/` (see [#134](https://github.com/makandra/spreewald/issues/134))
+  - `/^I should get a text response$/` (see [#135](https://github.com/makandra/spreewald/issues/135))
+  - `/^I wait for the page to load$/` (see [#136](https://github.com/makandra/spreewald/issues/136))
+
```

Comment:
`Given the file "..." was attached( as (.../)?...)? to the ... above( at "...")?` (the only step from file_attachment_steps` should be deprecated as well

Reply:
`Then the "..." checkbox should( not)? be checked( and disabled)?` should receive a deprecation for `and disabled`, see https://github.com/makandra/spreewald/pull/148

Reply:
The step from file_attachment_steps has been deprecated since 2.3.0, so I didn't deprecate it again.
As we discussed three weeks ago, `and disabled` is required in 2.99, as the step won't find disabled checkboxes without it.

Reply:
You are right, I forgot about the 2.3 deprecations.

Regarding the `and disabled` fragment I still think we should deprecate it in 2.99.
This gives the user the choice: Either go back to 2.9 or forward to 3.0. If we don't add the deprecation warning, we have to remove it without prior warning in 3.0